lib/sizzle.patched.js

```diff
@@ -1488,6 +1491,20 @@ Expr = Sizzle.selectors = {
 
 		"contains": markFunction(function( text ) {
```

Comment:
> Do you want to reassign the value of Sizzle.selectors.pseudos["contains"] somewhere else?

Yeah, outside of the Sizzle's code.

> Doesn't it require having duplicate codes?

I am okay with removing something from Sizzle (for instance their original contains implementation).

My point is that removing something is much easier to support than changing Sizzle's code. It might come in handy if they change Sizzle drastically in future.

> Also we need a referrence for markFunction.

Do we really need it? Why can't we simply use:
```
sizzle.selectors.pseudos["contains"] = sizzle.selectors.createPseudo
```